s4e-backend/src/main/java/pl/cyfronet/s4e/controller/request/CreateUserWithGroupsRequest.java

```diff
@@ -5,11 +5,18 @@
 
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotEmpty;
+import java.util.Set;
 
 @Data
 @Builder
 public class CreateUserWithGroupsRequest {
     @Email
     @NotEmpty
     private String email;
+    @NotEmpty
+    private String name;
+    @NotEmpty
+    private String surname;
+
+    private Set<String> slugsGroups;
```

Comment:
-> groupSlugs?

Reply:
and not groupsSlugs?

Reply:
I'd stick to my version, you only use the plural form on the last component usually